usr/share/rear/skel/NBKDC/etc/scripts/system-setup.d/90-start-nbkdc.sh

```diff
@@ -0,0 +1,46 @@
+
+source $VAR_DIR/recovery/nbkdc_settings
```

Comment:
If one of the `return` calls in the `prep` script fires then the file does not exist.

Unfortunately, if the admin will only notice this problem here during recovery then it is obviously too late to fix something. Maybe reworking the prep script to actually guarantee the existance of this file might be good to ensure reliable operations of ReaR with NBKDC.